support-frontend/assets/pages/paper-subscription-landing/components/tabs.jsx

```diff
@@ -38,27 +40,42 @@ type DispatchPropTypes = {|
 type PropTypes = {|
   ...StatePropTypes,
   ...DispatchPropTypes,
+  useDigitalVoucher: Option<boolean>,
 |};
 
+// This is a temporary workaround while we have both iMovo and vouchers
+// We can get rid of this when we drop vouchers
+const getTabTitle = (useDigitalVoucher, fulfilmentMethod) => {
+  if (fulfilmentMethod === 'HomeDelivery' || useDigitalVoucher) {
+    return tabs[fulfilmentMethod].name;
+  }
+  return 'Voucher Booklet';
```

Comment:
with this I guess I didn't explain my logic quite right, I just feel like if I was explaining to people 
I would say "if it's digital voucher but we're not using digital voucher, call it `Voucher Booklet` instead (other wise use the default name)"
I wouldn't say "if it's home delivery or we're using digital voucher use the default name, otherwise use `Voucher Booklet`" 
As home delivery and useDigitalVoucher feel like they're in different dimensions/unrelated to me so it's not clear why they're in the same if statement.
However, if that's not how you'd think of it, then keep it as is!  It's clearly correct logic even if not how I'd write it.